### DISCUSSION TITLE
Decorators are not enabled

### DISCUSSION THREAD

**Discussion by AdvaSoftware** [babel/babel#15582] [Q&A] [answered]
Hi. I'm using the following plugin and presets:

`{
  "presets": [["@babel/preset-env"]],
  "plugins": ["@babel/plugin-proposal-private-methods", ["@babel/plugin-proposal-decorators", { "version": "2023-01" }]]
}
`

And keep getting the following error when trying to build my project using npm:

```
Module build failed (from ./node_modules/babel-loader/lib/index.js):
SyntaxError: {FILE_NAME}: Decorators are not enabled.
If you are using ["@babel/plugin-proposal-decorators", { "version": "legacy" }], make sure it comes *before* "@babel/plugin-proposal-class-properties" and enable loose mode, like so:
	["@babel/plugin-proposal-decorators", { "version": "legacy" }]
	["@babel/plugin-proposal-class-properties", { "loose": true }]

```
I've tried using different versions of the decorators but to no avail. Also tried to debug the preset to detect plugin collisions but it never gives me any info.

Versions installed:
Dep:
"core-js": "^3.29.1",

DevDep:
"@babel/plugin-proposal-decorators": "^7.21.0",
"@babel/preset-env": "^7.20.2",
"@babel/core": "^7.21.3",

Thanks for the help!

**Comment by nicolo-ribaudo** [comment, marked as answer]
Hi, can you try with this config?
```json
{
  "presets": [["@babel/preset-env"]],
  "plugins": [
    ["@babel/plugin-proposal-decorators", { "version": "2023-01" }]
  ]
} 
```

The private methods transform is already included in `@babel/preset-env`.

**Reply by AdvaSoftware** [reply]
It works!! Many thanks!
